src/seckrit.py

```diff
@@ -10,11 +10,22 @@
 
 
 def main(args):
+
     # Load the YAML file that tells us which secrets we want to load (and what to do with them).
-    manifest = load_manifest(args.manifest_file)
+    if args.manifest:
+        manifest = load_manifest(args.manifest)
+    elif "SECKRIT_MANIFEST" in os.environ:
+        manifest = yaml.safe_load(os.environ.get('SECKRIT_MANIFEST'))
```

Comment:
We need to validate the manifest against the schema after loading it. The validator also adds values for certain fields in the manifest if they're missing and there is a default.

This is currently all handled in `load_manifest()`, but maybe that isn't the best name. We should probably do the loading and validating in their own functions.

I would suggest:
* `load_manifest_from_file(manifest_path)` - returns the manifest as a Python `dict` (i.e. `yaml.safe_load`)
* `load_manifest_from_string(manifest_string)` - same as above
* `validate_manifest(manifest)` - runs the manifest `dict` through the validator.

I would suggest renaming `